app/Http/Controllers/ContestController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Gladiator\Http\Controllers;
+
+use Gladiator\Models\Contest;
+use Gladiator\Http\Requests\ContestRequest;
+
+class ContestController extends Controller
```

Comment:
I think all our controllers are pluralized. So pluralize this one `ContestsController` and the name of the file ;)